### DISCUSSION TITLE
Solarized theme not displaying correctly

### DISCUSSION THREAD

**Discussion by MajorArchitect** [atom/atom#22504] [Q&A]
I freshly installed Atom the other day, and when I select the solarized theme it looks like this: 
![Screenshot from 2021-05-29 15-37-03](https://user-images.githubusercontent.com/61904023/120059278-cc956380-c093-11eb-9121-3efdd7fba63e.png)
I have tried uninstalling and reinstalling atom in a few different ways, including deleting all atom-related files from my home dir. I am on Ubuntu 20.04. Other themes seem to be working properly. Is anybody familiar with issues like this?

**Comment by chbk** [comment]
Yeah those variables appear red because they're scoped `constant.other` by the language-c grammar, which is just wrong. They shouldn't be `constant`, that scope is for literals. And `other` doesn't mean anything as a scope.

There's an open PR that addresses and fixes this: https://github.com/atom/language-c/pull/351
